.github/workflows/deploy.yml

```diff
@@ -57,4 +58,6 @@ jobs:
                 "EDLUsername=${EDL_USERNAME}" \
                 "EDLPassword=${EDL_PASSWORD}" \
                 "RtcGammaImage=${RTC_GAMMA_REPOSITORY}:${GITHUB_REF##*/}" \
-                "AuthPublicKey=${AUTH_PUBLIC_KEY}"
+                "AuthPublicKey=${AUTH_PUBLIC_KEY}" \
+                "DomainName=hyp3-${GITHUB_REF##*/}-api.asf.alaska.edu" \
```

Comment:
I'm not a huge fan of this, the prod api should just be hype-api not hype-master-api